README.txt

```diff
@@ -20,9 +20,35 @@ Pairs (AVPs) over TCP and S11 GTPV2C messages over UDP protocols.
         src/s1ap/conf/s1ap.json
         src/s11/conf/s11.json
         src/s6a/conf/*.json
+        src/s6a/conf/s6a_fd.conf
 
     c. update the certificates in src/s6a/conf/ with production one
 
+       1. update the Diameter Configuration File
+          In {install_path}/openmme/src/s6a/conf/s6a_fd.conf update the
+          following values accordingly:
+
+            Identity - The diameter identity. The default value is
+                       mme.localdomain
+            Realm - The diameter realm (everything past the first period
+                    of the Identity). The default value is localdomain
+
+       2. create Diameter Certificates
+
+       NOTE: The "Diameter Identity" is a fully qualified domain name that is
+       used to access the the Diameter peer. The "Diameter Host" is
+       everything up to the first period of the "Diameter Identity". The
+       "Diameter Realm" is everything after the first period of the
+       "Diameter Identity".
+
+       Execute the following commands, using the "Identity" configured in
+       {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+
+          cd {install_path}/openmme/src/s6a/conf
+          ./make_certs.sh <diameter_host> <diameter_realm>
+
+        For Example: ./make_certs.sh mme localdomain
+
```

Comment:
Current certificates are under src/mme-app/conf folder, src/mme-app/Makefile copies those certificates to target directory in 'install' make target. Either set location for generating certificates to src/mme-app/conf Or add certificate copy command in src/s6a/Makefile 'install' target.

Reply:
As certificates have been now removed, then point c. shall be done before point a. ?

Reply:
After looking at the makefiles and install target, it would be better if we follow the below changes (to make the approach cleaner).

1. Remove static certificates from openmme repo (already done as part of this PR)
2. Add make_certs.sh in src/s6a/conf (already done as part of this PR)
3. Update install target in src/s6a/Makefile to copy the make_certs.sh to the target (openmme/target/conf) directory
4. Remove copying of pem files in src/mme-app/Makefile install target
4. Update README.txt to run mme modules as below,
    Update the required conf files as required (point b in current README)
    make clean;make;make install 
    cd openmme/target/conf
    Generate the certificates
    cd openmme/target/
    ./run.sh
5. point c,d,e can be removed in README.txt
6. chmod +x run.sh, stop.sh (located in src/mme-app/) and push to openmme repo
7. In run.sh below line needs to be updated,
    ./bin/s6a > /dev/null & ==> ./bin/s6a-app > /dev/null &

Reply:
When this will be done, refactoring of Jenkins files is needed, as they currently follow current README and no make install is done.
```
e. start the mme modules in the following order,
        cd {install_path}/openmme/src/mme-app
        ./mme-app
        cd {install_path}/openmme/src/s1ap
        ./s1ap-app
        cd {install_path}/openmme/src/s11
        ./s11-app
        cd {install_path}/openmme/src/s6a
        ./s6a-app
```

Reply:
In run.sh, the starting order is different that current README. Is it intentional or irrelevant?
https://github.com/omec-project/openmme/blob/115757854784068b4d3ea5fab59a36f13cc1a481/src/mme-app/run.sh#L20-L27

Reply:
The order in run.sh and README both works fine. So it should not be a problem.